src/google/protobuf/compiler/objectivec/objectivec_file.cc

```diff
@@ -432,12 +432,9 @@ void FileGenerator::GenerateSource(io::Printer *printer) {
       "\n");
   if (!fwd_decls.empty()) {
     printer->Print(
-        "#pragma mark - Objective C Class references\n"
-        "// This somewhat arcane code forces linkage of classes from static archives by\n"
-        "// adding a concrete reference to the classes.\n"
-        "// We don't use `[Foo class]` because we need a static value for our initializer.\n"
```

Comment:
These last three lines might still be useful to explain to someone, up to you if you want to keep them.

Reply:
Added the important clarifying line IMHO.